src/components/Layout/Layout.component.jsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import MainNavigation from '../Navigation/MainNavigation';
 
 import './Layout.styles.css';
 
 function Layout({ children }) {
-  return <main className="container">{children}</main>;
+  return (
+    <>
+      <MainNavigation />
+      <main className="container">{children}</main>
```

Comment:
I recommend using only one pattern for styling: module-class or styling components. We require the project to use styled-components so we could review some patterns here. Even though we had class styles for the components, please refactor it to use styled-components.